test/chain/main.go

```diff
@@ -192,143 +199,6 @@ func playFaucetContract(chain *core.BlockChain) {
-	_, err = chain.InsertChain(types.Blocks{block}, true /* verifyHeaders */)
-	if err != nil {
-		fmt.Println(err)
-	}
-	// receipts := contractworker.GetCurrentReceipts()
-	// fmt.Println(receipts[len(receipts)-4].ContractAddress)
-	state = contractworker.GetCurrentState()
-
-	fmt.Printf("After Staking Balances (should be less by %d)\n", stake)
-	fmt.Println("user address balance")
-	fmt.Println(state.GetBalance(allRandomUserAddress[0]))
-	fmt.Println("The balances for 2 more users:")
-	fmt.Println(state.GetBalance(allRandomUserAddress[1]))
-	fmt.Println(state.GetBalance(allRandomUserAddress[2]))
-	fmt.Println("faucet contract balance (unchanged):")
-	fmt.Println(state.GetBalance(faucetContractAddress))
-	fmt.Println("stake contract balance :")
-	fmt.Println(state.GetBalance(stakeContractAddress))
-	fmt.Println("stake address balance :")
-	fmt.Println(state.GetBalance(StakingAddress))
-}
-
-func playWithdrawStaking(chain *core.BlockChain) {
-	fmt.Println()
-	fmt.Println("--------- Now Setting up Withdrawing Stakes ---------")
-
-	withdrawFnSignature := []byte("withdraw(uint256)")
-
-	fnHash := hash.Keccak256(withdrawFnSignature)
-	methodID := fnHash[:4]
-
-	withdraw := "5000"
-	withdrawstake := new(big.Int)
-	withdrawstake.SetString(withdraw, 10)
-	paddedAmount := common.LeftPadBytes(withdrawstake.Bytes(), 32)
-
-	var dataEncl []byte
-	dataEncl = append(dataEncl, methodID...)
-	dataEncl = append(dataEncl, paddedAmount...)
-
-	var withdrawstakingtxns []*types.Transaction
-
-	fmt.Println()
-	fmt.Printf("-- Withdrawing Stake by amount: %s --\n", withdraw)
-	fmt.Println()
-
-	for i := 0; i <= 2; i++ {
-		cnonce := contractworker.GetCurrentState().GetNonce(allRandomUserAddress[i])
-		tx, _ := types.SignTx(types.NewTransaction(cnonce, stakeContractAddress, 0, big.NewInt(0), params.TxGas*5, nil, dataEncl), types.HomesteadSigner{}, allRandomUserKey[i])
-		withdrawstakingtxns = append(withdrawstakingtxns, tx)
-	}
-
-	err = contractworker.CommitTransactions(withdrawstakingtxns, nil, common.Address{})
-	if err != nil {
-		fmt.Println("error:")
-		fmt.Println(err)
-	}
-
-	block, _ := contractworker.FinalizeNewBlock([]byte{}, []byte{}, 0, common.Address{}, nil, nil)
-	_, err = chain.InsertChain(types.Blocks{block}, true /* verifyHeaders */)
-	if err != nil {
-		fmt.Println(err)
-	}
-	state = contractworker.GetCurrentState()
-	fmt.Printf("Withdraw Staking Balances (should be up by %s)\n", withdraw)
-	fmt.Println(state.GetBalance(allRandomUserAddress[0]))
-	fmt.Println(state.GetBalance(allRandomUserAddress[1]))
-	fmt.Println(state.GetBalance(allRandomUserAddress[2]))
-	fmt.Println("faucet contract balance (unchanged):")
-	fmt.Println(state.GetBalance(faucetContractAddress))
-	fmt.Printf("stake contract balance (should downup by %s)\n", withdraw)
-	fmt.Println(state.GetBalance(stakeContractAddress))
-	fmt.Println("stake address balance :")
-	fmt.Println(state.GetBalance(StakingAddress))
-}
-
-func playStakingContract(chain *core.BlockChain) {
```

Comment:
why remove this test?

Reply:
These are old test for  the previous staking logic implemented via smart contract. It's deprecated.